doc/news/_preparation_next_release.md

```diff
@@ -223,6 +232,20 @@ Thanks to Michael Zronek and Vanessa Kos.
 
 [#1887]: https://github.com/ElektraInitiative/libelektra/issues/1887
 
+## Compatibility
```

Comment:
I think the "Compatibility" above happened at some wrong merge. @sanssecours any idea how this happened?

Reply:
I think the duplicated header was added in commit 1b2d0d62. Anyway, pull request [#2066](https://github.com/ElektraInitiative/libelektra/pull/2066/files#diff-47ed151a0df6c926f5b1320d140a5b34L227) should fix the problem.